tests/e2e/fixtures/statsFixtures.js

```diff
@@ -18,22 +18,21 @@ const seasonFixtures = [
     startTime: dayjs('2022-02-16 18:00').valueOf(),
     endTime: dayjs('2022-03-16 18:00').valueOf(),
   },
+  {
+    name: 'Current Season',
+    startTime: dayjs().subtract(2, 'week').valueOf(),
+    endTime: dayjs().add(12, 'week').valueOf(),
```

Comment:
This is right, but let's make it 13 weeks long since that's what we actually do for the competitive seasons
```suggestion
    endTime: dayjs().add(11, 'week').valueOf(),
```